src/Uno.UI/UI/Xaml/Controls/Image/Image.Android.cs

```diff
@@ -471,13 +305,13 @@ private async Task SetSourceResource(ImageSource newImageSource)
 
 			BitmapFactory.Options o = new BitmapFactory.Options();
 			o.InJustDecodeBounds = true;
-			Bitmap bmp = BitmapFactory.DecodeResource(Resources, newImageSource.ResourceId.Value, o);
+			Bitmap bmp = BitmapFactory.DecodeResource((this as View).Resources, newImageSource.ResourceId.Value, o);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused local variable 'bmp'.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5554394)